src/shout_subgroup/modify_subgroup.py

```diff
@@ -88,25 +99,34 @@ async def _handle_add_users_to_existing_subgroup(
         db: Session,
         update: Update,
         subgroup_name: str,
-        usernames: set[str]
+        users_ids_and_mentions: set[UserIdMentionMapping]
```

Comment:
Focus: Haffi use a mapping between the mention and user ids instead of user ids alone